strongbox-storage/strongbox-storage-api/src/main/java/org/carlspring/strongbox/ext/jersey/CustomJerseyHeaderDelegateProviders.java

```diff
@@ -0,0 +1,32 @@
+package org.carlspring.strongbox.ext.jersey;
+
+import javax.inject.Singleton;
+
+import org.glassfish.hk2.utilities.binding.AbstractBinder;
+import org.glassfish.jersey.message.internal.*;
+import org.glassfish.jersey.spi.HeaderDelegateProvider;
+
+/**
+ * @author Przemyslaw Fusik
+ *
+ * @see MessagingBinders.HeaderDelegateProviders
+ */
+public class CustomJerseyHeaderDelegateProviders
```

Comment:
I think this class should be under the `strongbox-client`. Any thoughts on this?

Reply:
I put it in the `strongbox-storage-api` because it was somehow related with `ProxyRepositoryProvider`. I think you're right it better suits to `strongbox-client`.